config/ember-try.js

```diff
@@ -2,12 +2,6 @@
 module.exports = {
   scenarios: [
     {
-      name: 'default',
```

Comment:
Good point, probably don't need that.  Can you update `.travis.yml` to eliminate that test run as well?